server/boot/user.js

```diff
@@ -249,7 +249,9 @@ module.exports = function(app) {
   }
 
   function getSettings(req, res) {
-    res.render('account/settings');
+    res.render('account/settings', {
+        title: 'Settings | Free Code Camp'
```

Comment:
Change `title: 'Settings | Free Code Camp'` to `title: 'Settings '`

The tool tip shows an extra `| Free Code Camp` :

![1](https://cloud.githubusercontent.com/assets/1884376/13845201/6242826e-ec64-11e5-9510-a71081663be4.png)